tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat.Tests - Copy/EntrypointResolverTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Linq;
+using Autofac.Extras.Moq;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Solution.Tests
+{
+    public class EntrypointResolverTests
+    {
+        [InlineData(new string[] { }, new string[] { }, new string[] { })]
+        [InlineData(new[] { "file1" }, new[] { "*" }, new[] { "file1" })]
+        [InlineData(new[] { "file1" }, new[] { "f" }, new string[] { })]
+        [InlineData(new[] { "file1" }, new[] { "i*" }, new string[] { })]
+        [InlineData(new[] { "file1" }, new[] { "i*", "f*" }, new[] { "file1" })]
+        [InlineData(new[] { "file1", "other" }, new[] { "i*", "f*" }, new[] { "file1" })]
+        [InlineData(new[] { "file1", "other" }, new[] { "o*", "f*" }, new[] { "file1", "other" })]
```

Comment:
I would add a test case where `fileNames` is not empty but `globs` is.

Reply:
Good catch